### ISSUE TITLE
Git submodules

### ISSUE THREAD

**Issue by irvinlim** [Precogs-com/lambda-boilerplate#1]
Hi, author of [`es2017-lambda-boilerplate`](https://github.com/irvinlim/es2017-lambda-boilerplate) here, glad to see that you created a generator for my boilerplate!

Just curious - what was your use case to create this generator? Also, why not [Yeoman](http://yeoman.io/) (though I have no experience using it either)?

Either way, I would prefer if you made use of either Git submodules, or `git clone` directly from the GitHub repo URL, instead of copying straight from source! It would help to aid updates as and when I publish new releases, and also allow users to install a specific version of the boilerplate.

Thank you for your support!

**Comment by baipi** [comment]
Hi @irvinlim,

Thank you for your comment, and your very complete boilerplate. You're right, we should have had submodules instead of ugly copy. Really sorry about that, I just updated the code and NPM package.

We built this command to have an easy light way to generate our base code.
Our infrastructure is based on Lambda, with some specific tools to fit our system. All our Lambda are based on several different templates, instead of duplicate code from an other Lambda, replacing code by the new one, we created templates (containing our specific mandatory libs, tests, linter, Cloudformation, ...). This command is just an easy way to pick a template, copying it at the right place and replacing some variables with the right value.

I didn't know Yeoman either, but it looks overweening and harder to user for our needs: using personal private template and integration in development flow.

Thanks again for your comment,

Have a nice day
